src/Elcodi/Bundle/AttributeBundle/ElcodiAttributeBundle.php

```diff
@@ -48,4 +49,16 @@ public function getContainerExtension()
     {
         return new ElcodiAttributeExtension();
     }
+
+    /**
+     * Create instance of current bundle, and return dependent bundle namespaces
+     *
+     * @return array Bundle instances
+     */
+    public static function getBundleDependencies()
+    {
+        return [
+            '\Elcodi\Bundle\CoreBundle\ElcodiCoreBundle',
```

Comment:
Can't wait to start supporting PHP >= 5.5 to make this like `ElcodiCoreBundle::class`